docs/openbanking/par-jarm.md

```
@@ -1,16 +1,16 @@
-This section covers details of two important features required by the open banking ecosystem. The latest Gluu Open Banking Identity Platform 
-supports [PAR](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-par-07) and [JARM](https://openid.net/specs/openid-financial-api-jarm-ID1.html) specifications.  
-These two features are bundled in the installation so when you install the Gluu Open Banking Identity Platform the Authorization 
-Server(AS) will support these features by default. The older/existing installation may require updating the WAR/ image. 
+This section covers details of two important features required by the open banking ecosystem. The latest Gluu Open Banking Identity Platform
+supports [PAR](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-par-07) and [JARM](https://openid.net/specs/openid-financial-api-jarm-ID1.html) specifications.
+These two features are bundled in the installation so when you install the Gluu Open Banking Identity Platform the Authorization
+Server(AS) will support these features by default. The older/existing installation may require updating the WAR/ image.
 
 Moreover, these features are also [FAPI certified for Brazil Open Banking (Based on FAPI 1 Advanced Final)](https://openid.net/certification/#FAPI_OPs).
 
 ## Pushed Authorization Requests-PAR:
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)